go_password_generator_cli/README.md

```diff
@@ -0,0 +1,14 @@
+# PROJECT TITLE -> Go_Password_Generator_CLI
```

Comment:
`Go_Password_Generator_CLI`

you can change it into this , replace PROJECT TITLE